var/spack/repos/builtin/packages/dftbplus/package.py

```diff
@@ -0,0 +1,140 @@
+from spack import *
+
+import os
+
+
+class Dftbplus(MakefilePackage):
+    """DFTB+ is an implementation of the
+    Density Functional based Tight Binding (DFTB) method,
+    containing many extensions to the original method."""
+
+    homepage = "https://www.dftbplus.org"
+    url      = "https://www.dftbplus.org/fileadmin/DFTBPLUS/public/dftbplus/19.1/dftbplus-19.1.tar.xz"
+
+    version('19.1', sha256='78f45ef0571c78cf732a5493d32830455a832fa05ebcad43098895e46ad8d220')
+
+    variant('mpi', default=True,
+            description="Build an MPI-paralelised version of the code.")
+
+    variant('gpu', default=False,
+            description="""Use the MAGMA library
+            for GPU accelerated computation""")
+
+    variant('elsi', default=False,
+            description="""Use the ELSI library for large scale systems.
+            Only has any effect if you build with '+mpi'""")
```

Comment:
This will probably end up with extra indentation and new-line characters in the `spack info` output. Instead of using something like:
```python
foo="""...
       ..."""
```
Since it's within parentheses, Python will automatically concatenate them. Just remember to add a space to the end of each line.
you can use:
```python
foo="..."
    "..."
```